### ISSUE TITLE
Use Greek symbols for genes in genotype table on management page

### ISSUE THREAD

**Issue by kimrutherford** [pombase/canto#1868]
Currently it shows gene names like "beta4GalNAcTB" instead of "β4GalNAcTB".

**Comment by kimrutherford** [comment]
> Currently it shows gene names like "beta4GalNAcTB" instead of "β4GalNAcTB".

This is a bit trickier than I thought.  The name we show is from the FlyBase Chado feature table and the "beta" is just plain text and not encoded in the same way as the allele names (with `&bgr;`).  I should have noticed that.

Because the Greek characters aren't encoded in the same way in the two cases we can't just reuse the allele display code.  So there's a bit more work than expected.  Is this worth spending time on?  Maybe we should leave it for now and come back to it once the high priority stuff is done?

**Comment by ValWood** [comment]
Do it when the  tickets for making useable are done (unless you run out of tickets before we discuss interactions!)

**Comment by vmt25** [comment]
I agree_ that is very low priority.

(anyways, leaving here a naive suggestion: given that FlyBase's allele symbols already include the gene symbol, could these by copied from the allele name by deleting bit in square brackets?)

**Comment by vmt25** [comment]
Eliminating the gene symbol column (see #1914 ) would solve this issue.
